src/main/resources/site/parts/article/article.es6

```diff
@@ -30,7 +30,7 @@ function renderPart(req) {
   moment.locale(page.language ? page.language : 'nb')
 
   const bodyText = processHtml({
-    value: page.data.articleText
+    value: page.data.articleText ? page.data.articleText.replace(/.&nbsp;/g, ' ') : undefined
```

Comment:
Any need for the "." in-front of "&nbsp;" here? It's not used the other places we need to replace nbsp